CMP1903_A1_2324/Testing.cs

```diff
@@ -8,13 +9,19 @@ namespace CMP1903_A1_2324
 {
     internal class Testing
     {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
 
-        //Method
+        public void Tests()
+        {
+            Game game = new Game();
+            int sum = game.sumGame();
```

Comment:
Object initializers use a 2 space indent well throughout the entire code, good consistency for following the Google Style Guide and making it easier to read.